packages/react-components/react-drawer/src/components/DrawerOverlay/DrawerOverlay.types.ts

```diff
@@ -1,16 +1,37 @@
-import { DialogProps, DialogSurfaceProps, DialogSurfaceSlots } from '@fluentui/react-dialog';
+import type { DialogProps } from '@fluentui/react-dialog';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 import type { MotionState } from '@fluentui/react-motion-preview';
 
 import type { DrawerBaseProps, DrawerBaseState } from '../../shared/DrawerBase.types';
+import { DrawerOverlaySurfaceProps } from './DrawerOverlaySurface';
 
-export type DrawerOverlaySlots = DialogSurfaceSlots & {
-  root: Slot<DialogSurfaceProps>;
+/**
+ * DrawerOverlay slots
+ */
+export type DrawerOverlaySlots = {
+  /**
+   * Slot for the root element.
+   */
+  root: Slot<DrawerOverlaySurfaceProps>;
+
+  /**
+   * Dimmed background of drawer.
+   * The default backdrop is rendered as a `<div>` with styling.
+   * This slot expects a `<div>` element which will replace the default backdrop.
+   * The backdrop should have `aria-hidden="true"`.
+   *
+   */
+  backdrop?: Slot<'div'>;
```

Comment:
```suggestion
```

I believe you don't need this as `backdrop` will be provided by `DrawerOverlaySurfaceProps` which you're merging into props by declaring it as root

Reply:
I still need this in case someone wants to override the backdrop.